packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import composeClasses from '@mui/utils/composeClasses';
 import { styled, useThemeProps, SxProps, Theme } from '@mui/material/styles';
-import { Popper, PopperProps as BasePopperProps } from '@mui/base/Popper';
-import { NoSsr } from '@mui/base/NoSsr';
-import { useSlotProps } from '@mui/base/utils';
+import Popper, { PopperProps as BasePopperProps } from '@mui/material/Popper';
```

Comment:
The current plan is to avoid @mui/material` usage as much as possible with the plan to decouple from it eventually completely.
What would be the next step for these usages? Importing from `@base-ui/react`?

Reply:
> What would be the next step for these usages? Importing from @base-ui/react?

Yes, once they are available. However, this change may contain breaking changes, and we can't do it immediately. I moved these imports for now, as any potential fixes will be done in the Material UI package from now on.

Reply:
Is their a difference between the `@mui/base` and `@mui/material` Popper ?

Reply:
The only difference is that the Mateiral UI Popper allows overrides via the theme (default props, style customization etc). Considering the X charts don't have dependency on Material UI, maybe it's better to keep the dependency on @mui/base, the downside is that if we find a bug, it will be complicate the release a fix, but if it is a deal breaker, we can keep maintaining it until Base UI has an alternative for it. I think this may ideal as otherwise we need to add @mui/material as a peer dependency on the charts which is not ideal.

Reply:
That's good for me to move to the material one. The extraction of the charts from the material is not complete, so that diff is not a big deal